stand/efi/loader/copy.c

```diff
@@ -380,7 +381,7 @@ efi_check_space(vm_offset_t end)
 	end += staging_slop;
 
 	nr_pages = EFI_SIZE_TO_PAGES(end - staging_end);
-#if defined(__amd64__)
+#if defined(__amd64__) || defined(__i386__)
 	/*
 	 * amd64 needs all memory to be allocated under the 1G or 4G boundary.
```

Comment:
This coment needs to be adjusted

Reply:
I interpreted that as "the amd64 kernel needs all memory ...", as the above comments were also taking about the kernel.